dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -43,7 +43,7 @@ function setVars() {
 #    export RH_CHE_DOCKER_IMAGE_REGISTRY="dfestal/che-server";
     export RH_CHE_PROJECT_ID="67da9ee-fabric8-ce16ce1";
     export RH_CHE_DOCKER_IMAGE_REGISTRY="rhche/rh-che-server";
-    export RH_CHE_PROJECT_NAMESPACE=che6-automated;
+    export RH_CHE_PROJECT_NAMESPACE=dfestal-che;
```

Comment:
@ScrewTSW I had to use my namespace here, because, for an unknown reason, your script didn't correctly create the `che6-automated` namespace in my dev cluster account, nor switched to it : so everything was inconsistent. I had to create the namespace and change it here in your script.
Seems to me this requires at least:
- checking that `oc` is effectively using the expected  `RH_CHE_PROJECT_NAMESPACE` before applying further `oc` commands
- allowing the user to override the OS namespace in which everything should be created

If you fix this in another PR, I'll change it here and will rebase my PR on your

Reply:
Will look into that, but I'm suspecting that the namespace (if created before) might have been incorrectly deleted by openshift.

Reply:
Radim suggested and I think he's right, the issues is with "You do not have access to this namespace" which is probably incorrectly handled by the CLI `origin-clients` package.

Basically, you cannot have namespaces with the same name, and it failed for you, because I already had the namespace created in my account, tied to my user.

Is there a way that origin would allow to create multiple namespaces with the same name?

I will modify the script so that the `cheX-automation` is used by default, but you will be able to enter a custom name through a flag